src/@types/blog.d.ts

```diff
@@ -0,0 +1,10 @@
+export interface BlogPostTeaser {
```

Comment:
this is shared across two locations so I put them in here, I'm not sure how to handle shared types in general right now.

Reply:
`@types` folder is for global definition files, isn't it? so not for "regular shared interfaces", but I would need to dig deeper into that topic to make a technically correct statement. For now it's ok to have it here.

In other projects, we are using `src/types.ts`.

Reply:
oh you are right. I had some weird feeling doing so. I will create `types.ts`